scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl

```diff
@@ -18,59 +18,44 @@ workflow GvsCalculatePrecisionAndSensitivity {
   parameter_meta {
     input_vcfs: "A collection of VCFS used for analysis (these are generated by `GvsExtractCallSet'). Note: these need not be subsetted down to the chromosome."
     output_basename: "The base name for the output files generated by the pipeline."
-    chromosome: "The chromosome on which to run the analysis of Precision and Sensitivity. The default value for this is `chr20`. If it is set to `all` then the analysis will be run across *all* chromosomes."
+    chromosomes: "The chromosome(s) on which to analyse precision and sensitivity with RTG tools. The default value for this is `['chr20']`."
```

Comment:
🇺🇸 
```suggestion
    chromosomes: "The chromosome(s) on which to analyze precision and sensitivity with RTG tools. The default value for this is `['chr20']`."
```

Reply:
Will the expected audience know what RTG tools are? I did not 😞

Reply:
I mean, it's what the workflow does.  But I don't necessarily have to expose that here.